src/index.ts

```diff
@@ -318,7 +326,7 @@ export function startHeapProfiling(): void {
     const profile = pprof.heap.profile(undefined, config.sm)
     log('Heap profile collected...')
     uploadProfile(profile).then(() => log('Heap profile uploaded...'))
-  }, INTERVAL)
+  }, Number(SAMPLING_DURATION))
```

Comment:
Ideally, we need one more parameter, like `UPLOAD_INTERVAL`, which would allow scenarios like "profile my app for 10s every minute", but we can implement it later